### DISCUSSION TITLE
β版経由での更新と予告無しでの突然の更新。この使い分けは？

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#336] [要望・アイデア・不具合]
最近行われたエディタ機能の更新では、正規の更新に先行してベータ版での試用の機会が提供され、ユーザーが評価結果や改善の要望をコメントする場もありました。

　しかし、先日の検索機能の更新は、予告もなく突然行われました。しかも、更新前後での使い勝手の変化はかなり大きいように感じます。「>」ボタンと「>>」ボタンの間隔を広げる程度の軽微な変更であれば、予告などは必要ないと思いますが、今回の更新は軽微とは思えません。

　どのような方針で使い分けておられるのかご教示ください。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
前提として、元々検索ページのデザインが古いままだったということがあるのですが
別の目的で検索ページのアップデートを行った際に、他ページとデザインが統一されて、現在のデザインに変更されました🙏

多くのユーザーさまに影響があると考えられる、大きな機能のアップデートではβ版をリリースをしておりますが
影響が小さいと判断したものはβ版を使わずに、リリースをすることもあります。
今回は、元々他ページでは既に変わっていたデザインに統一される形での修正だったため、予告も不要と判断しました。

ページネーションの構成は他のページでの検証の上で今の作りになっています。
今回のようにご希望に沿えないこともあるかと思いますが、ご理解いただけると幸いです🙇
